BUS-22 route picker

TravMurav opened the pull request.

TravMurav committed:
templates: route picker

TravMurav committed:
templates: list of found routes

almiluk reviewed (changes requested):
Непонятные имена файлов шаблонов, надо бы изменить

TravMurav committed:
templates: give sane names to the files

WHO COULD"VE KNEW that the ABI for the funny files that looked like
this:

def routes_page():
    return render_template("find_routes.html")

def find_routes(route_data):
    return render_template("routes.html")

would MAGICALLY disappar...

almiluk commented:
Don't know what are your talking about
